### PR TITLE
Added Fuse release 7.8

### PR DESCRIPTION
**Pull request by valdar**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -107,6 +107,10 @@
           "id": "community",
           "name": "2.21.2 (Community)"
```
Comment (apupier) [inline metadata.json:108]: shouldn't we add a/update to 2.23.x community version to match the 7.8 productized version?

**Comment by quintesse** [comment]
[test]

**Comment by quintesse** [comment]
[test]

**Comment by quintesse** [comment]
[test]

File: metadata.json (hunk 1 above)
Comment (valdar) [reply metadata.json:108]: Well this is fuse 7.8 release, we can surely open another PR to bump community version as well...

**Comment by quintesse** [comment]
[test]

**Comment by quintesse** [comment]
Is this ready to be merged?

**Comment by quintesse** [comment]
Never got feedback on this, can this be merged?

**Comment by valdar** [comment]
@quintesse Sorry I must have missed a notification. You can merge this one thanks!

**Comment by quintesse** [comment]
Ok done! 👍

**Comment by valdar** [comment]
@quintesse  it seems it is not reflected in https://forge.api.openshift.io or https://launch.openshift.io/ i.e. I can not see fuse runtime version 7.8 deployed there... cc @apupier

**Pull request merged**